app/src/main/java/com/example/coronadiagnosticapp/ui/activities/oxymeter/OxymeterImpl.java

```diff
@@ -231,7 +225,7 @@ private void writeCSV(List<String[]> data) {
 
     @Override
     public OxymeterData finish(double totalTimeInSecs) {
-        double samplingFreq = (counter / totalTimeInSecs);
+        double samplingFreq = 30;
```

Comment:
There should be a way to confirm this from the camera metadata, some phones might support only 15fps.
Not a good idea to rely on all phones supporting 30FPS.